docs/adr/0003-file-based-data.md

```diff
@@ -0,0 +1,88 @@
+# Use files for metrics data
+
+- Status: rejected
+- Date: 2020 (this ADR was backfilled for historical purposes)
+
+## Context and Problem Statement
+
+The GLAM data consists mainly of a histogram, percentiles, and counts for each
+set of dimensions. This can be stored in a file whose path and name can
+represent the dimensions and whose content can contain the actual data.
```

Comment:
Worth mentioning the current postgres-based architecture. I recommend at least linking to your google doc:

https://docs.google.com/document/d/1JN4XQdG-8S5wbRUWAFp0XpDfqNu9ti_rdmy8OZXuEyE/edit

(that would in itself be a good ADR, though I understand if you don't have time to reformat it)